.forge-snapshots/simpleSwapEOAInitiated.snap

```diff
@@ -1 +1 @@
-172919
\ No newline at end of file
+173759
```

Comment:
This test is so weird... something is making the gas go wrong.
But its being removed in #480 so not going to dig into it